MagicCards/Podfile.lock

```diff
@@ -35,4 +35,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: a94c5794a207a915e624b269a8ee0eb2ec43fb01
 
-COCOAPODS: 1.9.0
+COCOAPODS: 1.7.5
```

Comment:
Era seu objetivo diminuir a versão do cocoa pods? Aconselho vcs usarem uma ferramenta como o Bundler e rbenv:
https://github.com/rbenv/rbenv
https://bundler.io

Reply:
@cs-adriel-freire @cs-carolina-lopes @cs-lucca-franca é realmente necessário upar para o GitHub o seu Pods/ folder? O ideal é só upar o seu Podfile e o lock correspondente, para evitar subir arquivos que vão inflar futuramente seu repo. #fica_a_dica

Reply:
Fui eu que tinha dito pra manter, pq eu li que no [guia do cocoaPods ](https://guides.cocoapods.org/using/using-cocoapods#should-i-check-the-pods-directory-into-source-control)eles recomendam manter a pasta de pods visível pro git.

Reply:
Mas acho que o a gente pode se reunir depois e ver essa questão de ignorar ou não novamente